doc/build-instructions/Build_Instructions_V8.md

```diff
@@ -159,10 +125,8 @@ bash get_source.sh
 :penguin:
 When you have all the source files that you need, run the configure script, which detects how to build in the current build environment.
 ```
-bash configure --with-freemarker-jar=/root/freemarker.jar
+bash configure --with-cmake
```

Comment:
We're not ready for this yet.

Reply:
I thought we were close enough (at least for jdk8 on Linux) that it made sense to update this, but I will revert to non-cmake for now.

Reply:
Reverted.